config/routes.rb

```diff
@@ -0,0 +1,7 @@
+Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
```

Comment:
- [ ] Great job working on this app. Kindly remove all pieces of commented code, as this will make your repo clean and professional.